python/pyspark/sql/tests/streaming/test_streaming_listener.py

```diff
@@ -381,7 +381,8 @@ def verify(test_listener):
                     .start()
                 )
                 self.assertTrue(q.isActive)
-                q.awaitTermination(10)
+                while progress_event is None or progress_event.batchId == 0:
```

Comment:
do we not want to add a timeout that could be longer? Maybe 60 seconds or something?